docs/get-started/developers/index.md

```diff
@@ -26,32 +89,178 @@ Read this section to get started and e-mail [design-system@redhat.com](mailto:de
+      <path d="M147.325 62.018L141.377 45.771" stroke="#A3A3A3" stroke-width="2" stroke-linejoin="bevel" stroke-dasharray="16 8"/>
+      <path d="M137.4 36.2001L147.133 44.2438L135.3 48.651L137.4 36.2001Z" fill="#A3A3A3"/>
+      <path d="M200.144 94.8159L253.764 65.6934" stroke="#A3A3A3" stroke-width="2" stroke-linejoin="bevel" stroke-dasharray="16 8"/>
+      <path d="M202.381 89.3156L210.499 74.0371" stroke="#A3A3A3" stroke-width="2" stroke-linejoin="bevel" stroke-dasharray="16 8"/>
+      <path d="M214.965 64.6836L215.48 77.2998L204.296 71.438L214.965 64.6836Z" fill="#A3A3A3"/>
+      <path d="M201.513 143.04L233.483 195.013" stroke="#A3A3A3" stroke-width="2" stroke-linejoin="bevel" stroke-dasharray="16 8"/>
+      <path d="M207.126 144.977L222.82 152.26" stroke="#A3A3A3" stroke-width="2" stroke-linejoin="bevel" stroke-dasharray="16 8"/>
+      <path d="M232.4 156.215L219.83 157.41L225.081 145.927L232.4 156.215Z" fill="#A3A3A3"/>
+      <path d="M94.8564 94.8159L41.2356 65.6934" stroke="#A3A3A3" stroke-width="2" stroke-linejoin="bevel" stroke-dasharray="16 8"/>
+      <path d="M92.6193 89.3156L84.5011 74.0371" stroke="#A3A3A3" stroke-width="2" stroke-linejoin="bevel" stroke-dasharray="16 8"/>
+      <path d="M80.0355 64.6836L79.5202 77.2998L90.7037 71.438L80.0355 64.6836Z" fill="#A3A3A3"/>
+      <path d="M93.4873 143.04L61.5168 195.013" stroke="#A3A3A3" stroke-width="2" stroke-linejoin="bevel" stroke-dasharray="16 8"/>
+      <path d="M87.874 144.977L72.1801 152.26" stroke="#A3A3A3" stroke-width="2" stroke-linejoin="bevel" stroke-dasharray="16 8"/>
+      <path d="M62.5999 156.215L75.1699 157.41L69.9194 145.927L62.5999 156.215Z" fill="#A3A3A3"/>
+      <path d="M152.769 175L138.625 234.357" stroke="#A3A3A3" stroke-width="2" stroke-linejoin="bevel" stroke-dasharray="16 8"/>
+      <path d="M155.368 180.339L161.316 196.586" stroke="#A3A3A3" stroke-width="2" stroke-linejoin="bevel" stroke-dasharray="16 8"/>
+      <path d="M165.293 206.157L155.56 198.113L167.393 193.706L165.293 206.157Z" fill="#A3A3A3"/>
+    </g>
+  </svg>
   <div>
-    <h3>Foundations</h3>
-    <p><a href="/foundations">Foundations</a> are how we express our brand through color, space, typography, etc.</p>
-  </div>
-  <div>
-    <h3>Design tokens</h3>
-    <p><a href="tokens">Design tokens</a> are how we translate our design language decisions into code.</p>
+
+#### Encapsulation
+
+Web Components encapsulate their templates, styles, and behaviour. They establish a strong boundary between the component's internals and the rest of the website, letting developers write more modular code while avoiding conflicts with page-level CSS and JS. This hides complex logic and templating from the rest of the page's UI, which helps to ship design systems and applications at scale. In addition to encapsulating styles, Web Components can ship complex accessibility patterns, making it easier for page authors to provide accessible rich experiences.
```

Comment:
I think it should be "accessible, rich experiences" at the very end of this paragraph.